manifests/m/monkeyWie/Gopeed/1.7.0/monkeyWie.Gopeed.installer.yaml

```
@@ -13,7 +13,7 @@ ProductCode: '{5960F34D-1E42-402C-8C85-DE2FF24CBAE4}_is1'
 ReleaseDate: 2025-04-08
 Installers:
 - Architecture: x64
-  InstallerUrl: https://github.com/GopeedLab/gopeed/releases/download/v1.7.0/Gopeed-v1.6.12-windows-amd64.zip
+  InstallerUrl: https://gopeed.com/api/download?tpl=Gopeed-v1.7.0-windows-amd64.zip
   InstallerSha256: CD0250EFE3016BBC8FEFC212A2338B5CEE0C7D20B5D55CD1DA4D0A9BAC7C8567

```

```suggestion
  InstallerSha256: A553E448BB9BEFEC413B4AE2407D8792C2F1196E9598003E1CE4A76848AFB0F1
```

Hash by https://github.com/DuckDuckStudio/winget-tools/actions/runs/14516507278/job/40726716211#step:8:16